components/Style/Font.styled.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+const StyledH1 = styled.h1`
+  margin: 0;
+  font-weight: 700;
+  font-size: 20px;
+`;
+
+const StyledH2 = styled.h2`
+  font-size: 20px;
+  margin: 20px 25px 10px 25px;
+`;
+
+const StyledH3 = styled.h3`
+  margin: 10px 0 5px 20px;
+  font-size: 16px;
+`;
+
+const StyledH4 = styled.h4`
+  font-size: 16px;
+  margin: 60px 0 0 0;
+`;
```

Comment:
- If you always want to use these values for the respective headlines, you could think about putting them in the `GlobalStyles.js`. That would be a good idea for the `<h1>`, for example
- Here too, I would like to recommend that you choose a relative unit for the` font sizes`.

```suggestion
const StyledH1 = styled.h1`
  margin: 0;
  font-weight: 700;
  font-size: 1.2rem;
`;

const StyledH2 = styled.h2`
  font-size: 1.2rem;
  margin: 20px 25px 10px 25px;
`;

const StyledH3 = styled.h3`
  margin: 10px 0 5px 20px;
  font-size: 1rem;
`;

const StyledH4 = styled.h4`
  font-size: 1rem;
  margin: 60px 0 0 0;
`;
```